### PR TITLE
Add persist changes in release notes

### PR DESCRIPTION
**Pull request by daneshk**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -54,8 +54,24 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 - Added support for GraphQL interceptor configurations.
 - Added support to access subfields of a GraphQL field from the `graphql:Field` object.
 
+#### `persist` package
+
+- Added support for `in-memory` data store.
```
Comment (daneshk) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:59]:
```suggestion
- Added support for `In-Memory` data store.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Persist Tool
+
+- Added support for `in-memory` data store. Set as the default data store.
```
Comment (daneshk) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:95]:
```suggestion
- Added support for the `In-Memory` data store. Set as the default data store.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
...
+- Added support for `Google Sheets` data store. The support for Google Sheets data store is currently an experimental feature, and its behavior may be subject to change in future releases.
```
Comment (daneshk) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:96]:
```suggestion
- Added support for the `Google Sheets` data store. The support for Google Sheets data store is currently an experimental feature, and its behavior may be subject to change in future releases.
```

**Review by kaneeldias — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -54,8 +54,24 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 - Added support for GraphQL interceptor configurations.
 - Added support to access subfields of a GraphQL field from the `graphql:Field` object.
 
+#### `persist` package
+
+- Added support for the `In-Memory` data store.
+- Added support for the `Google Sheets` data store. The support for Google Sheets data store is currently an experimental feature, and its behavior may be subject to change in future releases.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:60]:
```suggestion
- Added support for the `Google Sheets` data store. This is currently an experimental feature and its behavior may be subject to change in future releases.
```
Should it be `Google Sheets` data store or `google-sheets` data store?

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 5]
```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Persist Tool
+
+- Added support for the `In-Memory` data store. Set as the default data store.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:95]:
```suggestion
- Added support for the `In-Memory` data store. This is set as the default data store.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 6]
```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
...
+- Added support for the `Google Sheets` data store. The support for Google Sheets data store is currently an experimental feature, and its behavior may be subject to change in future releases.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:96]:
```suggestion
- Added support for the `Google Sheets` data store. This is currently an experimental feature and its behavior may be subject to change in future releases.
```
Why is this repeated above also?

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 7]
```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
...
+- Added `persist migrate` command to migration scripts for data model changes in the `mysql` data store. The support for migrations is currently an experimental feature, and its behavior may be subject to change in future releases.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:97]:
```suggestion
- Added the `persist migrate` command to execute migration scripts for data model changes in the `mysql` data store. The support for migrations is currently an experimental feature and its behavior may be subject to change in future releases.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 8]
```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
...
+
 ### Improvements
 
+#### Persist Tool
+- Added support for additional DB configurations in MySQL generated client objects.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:102]:
```suggestion
- Added support for additional DB configurations in MySQL generated client objects.
```
Should it be `MySQL-generated`?

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 9]
```diff
@@ -74,8 +90,18 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
...
+- Changed the Ballerina `byte[]` type mapping to `LONGBLOB` in MySQL generated script file.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:103]:
```suggestion
- Changed the Ballerina `byte[]` type mapping to `LONGBLOB` in the MySQL generated script file.
```
Should it be `MySQL-generated`?

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 10]
```diff
@@ -54,8 +54,24 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
...
 
+#### `persist` package
+
+- Renamed the error types to the following:
+    - `InvalidKeyError` to `NotFoundError`
+    - `DuplicateKeyError` to `AlreadyExistsError`
+    - `ForeignKeyConstraintViolationError` to `ForeignKeyViolationError`
+- Removed the `FieldDoesNotExistError` error type as the `NotFoundError` error type can be used instead.
+- Marked the generated client object as `isolated`.
+- Added support for transactions in the `MySQL` data store.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:73]:
```suggestion
- Added support for transactions in the `mysql` data store.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 11]
```diff
@@ -54,8 +54,24 @@ To view bug fixes, see the [GitHub milestone for 2201.6.0 (Swan Lake)](https://g
 - Added support for GraphQL interceptor configurations.
 - Added support to access subfields of a GraphQL field from the `graphql:Field` object.
 
+#### `persist` package
+
+- Added support for the `In-Memory` data store.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:59]:
```suggestion
- Added support for the `in-memory` data store.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md (hunk 6 above)
Comment (daneshk) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:96]: We added this support in both module and tooling side

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md (hunk 8 above)
Comment (daneshk) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:102]: Better it change to 'generated MySQL client object. WDYT?

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md (hunk 7 above)
Comment (daneshk) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:97]: Should be generate not execute

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:97]: Updated.

**Pull request merged**